### DISCUSSION TITLE
Welcome to io_pdx_mesh Discussions page!

### DISCUSSION THREAD

**Discussion by ross-g** [ross-g/io_pdx_mesh#26] [General]
## Welcome! 👋
  I've enabled Github Discussions as a place to connect with other members of the Paradox/modding community.
Feel free to:
  * Ask questions you’re wondering about.
  * Share ideas or workflows.
  * Engage with other Github members.
 
Please welcome others and be open-minded.

Want to post a bug or feature request? Consider opening an [Issue](https://github.com/ross-g/io_pdx_mesh/issues) instead of starting a discussion.

**Comment by Yrrah2-byte** [comment]
Hello,

I am posting here because I have been unable to submit a question on the Paradox forum in the appropriate thread regarding this add-on for Blender.  The forum rejects any post of mine no matter how small and innocuous I make it, due to "spam-like contents" or some such.

At any rate I'm hoping this forum is still monitored.

The problem I'm having is that Blender 2.92 (and 2.91) will not activate this addon.  Here is the error I get:

Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 2.92\2.92\scripts\modules\addon_utils.py", line 382, in enable
    mod.register()
AttributeError: module 'io_pdx_mesh' has no attribute 'register'

I understand that the current addon is used in both Maya and Blender?  If so is there some setting I need to enable in order to use the Blender script?

Thanks for your help.

**Reply by ross-g** [reply]
Hi @Yrrah2-byte yes this forum is monitored 😄 thanks for checking out the tool. Unfortunately it didn't work with Blender from 2.91 for a while (see issue #24), but the problem you have is fixed in the [latest release here](https://github.com/ross-g/io_pdx_mesh/releases/latest).

Good luck!

**Comment by Yrrah2-byte** [comment]
Yes, that works now!

Thank you very much for this add-on and the quick response to my problem.

Best Regards

**Comment by AtomicCones** [comment]
Curious, i cant remember where i saw it, if it was elsewhere or from this plugin. But i know the Clausewitz engine has a limit of 50 bones or something?.. if that is correct, is there also a limit on polys / tri's??
